src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -107,164 +116,87 @@ class modify_commandscript : public CommandScript
         if (handler->HasLowerSecurity(target, ObjectGuid::Empty))
             return false;
 
-        handler->PSendSysMessage(LANG_YOU_CHANGE_HP, handler->GetNameLink(target).c_str(), hp, hpm);
-        if (handler->needReportToTarget(target))
-            ChatHandler(target->GetSession()).PSendSysMessage(LANG_YOURS_HP_CHANGED, handler->GetNameLink().c_str(), hp, hpm);
-
-        target->SetMaxHealth(hpm);
-        target->SetHealth(hp);
-
         return true;
     }
 
-    //Edit Player Mana
-    static bool HandleModifyManaCommand(ChatHandler* handler, const char* args)
+    //Edit Player HP
+    static bool HandleModifyHPCommand(ChatHandler* handler, const char* args)
     {
-        if (!*args)
-            return false;
-
-        int32 mana = atoi((char*)args);
-        int32 manam = atoi((char*)args);
-
-        if (mana <= 0 || manam <= 0 || manam < mana)
+        int32 hp, hpmax;
+        Player* target = handler->getSelectedPlayerOrSelf();
+        if (CheckModifyResources(handler, args, target, hp, hpmax))
```

Comment:
A good principle in software engineering is to terminate control flows early, which means:

```c++
if (!CheckModifyResources(handler, args, target, hp, hpmax))
    return false;
```

is preferrable over

```c++
if (CheckModifyResources(handler, args, target, hp, hpmax))
{
    ...
    return true;
}

return false;
```

You don't need to change this, I just mentioned it because it might help you in the future.